test/core/statistics/timers_test.c

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Can we move this to core/profiling perhaps?

Don't want to blur the line between this and census.